From 82935865af97da5b08ad0379d0763640753b0c7b Mon Sep 17 00:00:00 2001
From: Srikanth Thokala <srikanth.thokala@xilinx.com>
Date: Tue, 24 Sep 2013 20:00:51 +0530
Subject: [PATCH 456/628] axiethernet: xilinx: Fix kernel-doc warnings

git://github.com/Xilinx/linux-xlnx.git xilinx-v2013.4
commit 869d1f9da675244f16840fc151098b9cbd3e73b4

This patch remove kernel-doc warnings.

Signed-off-by: Srikanth Thokala <sthokal@xilinx.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
Signed-off-by: Liming Wang <liming.wang@windriver.com>
---
 drivers/net/ethernet/xilinx/xilinx_axienet_main.c |    3 +--
 1 files changed, 1 insertions(+), 2 deletions(-)

diff --git a/drivers/net/ethernet/xilinx/xilinx_axienet_main.c b/drivers/net/ethernet/xilinx/xilinx_axienet_main.c
index a22609f..5e260d5 100644
--- a/drivers/net/ethernet/xilinx/xilinx_axienet_main.c
+++ b/drivers/net/ethernet/xilinx/xilinx_axienet_main.c
@@ -1252,7 +1252,7 @@ axienet_ethtools_get_pauseparam(struct net_device *ndev,
  * axienet_ethtools_set_pauseparam - Set device pause parameter(flow control)
  *				     settings.
  * @ndev:	Pointer to net_device structure
- * @epauseparam:Pointer to ethtool_pauseparam structure
+ * @epauseparm:Pointer to ethtool_pauseparam structure
  *
  * This implements ethtool command for enabling flow control on Rx and Tx
  * paths. Issue "ethtool -A ethX tx on|off" under linux prompt to execute this
@@ -1507,7 +1507,6 @@ static void axienet_dma_err_handler(unsigned long data)
 /**
  * axienet_of_probe - Axi Ethernet probe function.
  * @op:		Pointer to platform device structure.
- * @match:	Pointer to device id structure
  *
  * returns: 0, on success
  *	    Non-zero error value on failure.
-- 
1.7.5.4

